### DISCUSSION TITLE
Error in packer_compiled: ...m/site/pack/packer/start/packer.nvim/lua/packer/load.lua:91: Vim:E492: Not an editor command: NvimTre eToggle Please check your config for correctness

### DISCUSSION THREAD

**Discussion by AdamBD** [nvim-tree/nvim-tree.lua#1894] [Q&A]
Anyone ever run into this issue?

Was working fine and one day stopped

**Comment by gegoune** [comment]
Seems like an installation issue? Did you try to recompile packer?

**Comment by alex-courtis** [comment]
Could it be a race condition around loading order and setup?

Perhaps you are invoking `:NvimTreeToggle` before setup has been called. Commands are registered during setup rather than on plugin load.
